README.md

```diff
@@ -107,7 +107,7 @@ The output will be something like the following:
 
 ![Hello World](https://github.com/textualize/rich/raw/master/imgs/hello_world.png)
 
-That's fine for styling a line of text at a time. For more finely grained styling, Rich renders a special markup which is similar in syntax to [bbcode](https://en.wikipedia.org/wiki/BBCode). Here's an example:
+That's fine for styling a line of text at a time. For more finely-grained styling, Rich renders a special markup that is similar in syntax to [bbcode](https://en.wikipedia.org/wiki/BBCode). Here's an example:
```

Comment:
Not sure the hyphenation is right here. Rules for adverb-adjective are, according to my understanding:

* Never hyphenate when the adverb ends in "ly"
* For other adverbs such as "well", there should be a hyphen if the adv-adj combo precedes a noun, but not if it doesn't. So "that's a well-worn trope", but "that trope is well worn".